content/spin/dynamic-configuration.md

```diff
@@ -135,3 +136,49 @@ Got password test_password
 
 Runtime configuration contains information for the selected config provider, such as the [Vault config provider](#vault-config-provider).
 You can supply runtime configuration by providing a value for the `--runtime-config-file` flag when invoking the `spin up` command.
+
+### Key Value Store Runtime Configuration
+
+Spin provides built-in key-value storage. This storage is backed by an SQLite database embedded in Spin by default. However, the Spin runtime configuration file (runtime-config.toml) can be updated to not only modify the SQLite configuration but also choose to use a different backing store. The only store options are the embedded SQLite database or an external Redis database. The following is an example of how an application's `runtime-config.toml` file can be configured for the single default key-value store option.
+
+```toml
+
+[key_value_store.default]
```

Comment:
I am begging you to remove this code block (and the last sentence of the preceding paragraph).  Users should never, ever see or write this.  It can only cause confusion.